frontend/src/app/Settings/Users/ManageUsers.tsx

```diff
@@ -262,7 +259,8 @@ const ManageUsers: React.FC<Props> = ({
       });
       const fullName = displayFullName(firstName, "", lastName);
       showSuccess("", `User name changed to ${fullName}`);
-      await queryUserWithPermissions();
+      updateLocalUserState("firstName", firstName);
```

Comment:
I think we might want to refetch instead of just updating the local state because it doesn't update in the UI after changing it. I did notice that on prod there is an existing issue of the name in the sidebar not being updated after a name update as well so this it not introduced in this PR.

https://github.com/CDCgov/prime-simplereport/assets/20211771/a03297e3-6782-49e0-8314-8ff604c15a80

Reply:
plus one to this... I would like to retain the logic to refetch the user instead of just updating the local state

Reply:
I'll fix that and just fetch instead of update local user, but weirdly enough updating the state works for last name and not first name 🤔